src/components/PathwayCourse/index.js

```diff
@@ -106,36 +146,156 @@ function PathwayCourse() {
-                    Learning Track
-                  </Typography>
-                  <Typography
-                    variant="h4"
-                    className={classes.heading}
-                    sx={{ textAlign: isActive && "center", pb: "16px" }}
+            <PathwayCards
+              userEnrolledClasses={userEnrolledClasses}
+              data={data}
+            />
+          </>
+        ) : (
+          pathwayId &&
+          pathwayCourseData && (
+            <>
+              <Grid
+                container
+                spacing={2}
+                align="center"
+                className={classes.box}
+              >
+                <Grid item xs={12} md={6} sx={{ pl: 2 }}>
+                  <Card
+                    align="left"
+                    elevation={0}
+                    className={classes.titleCard}
                   >
-                    {pathwayCourseData.pathway}
-                  </Typography>
-                  <Typography variant="body1">
-                    {pathwayCourseData.description}
-                  </Typography>
-                </Card>
-              </Grid>
-              {/* <Grid xs={12} md={6} sx={{ pl: 2 }}>
+                    <Typography
+                      variant="body2"
+                      className={classes.cardSubtitle}
+                      sx={{ textAlign: isActive && "center", pb: "8px" }}
+                    >
+                      Learning Track
+                    </Typography>
+                    <Typography
+                      variant="h4"
+                      className={classes.heading}
+                      sx={{ textAlign: isActive && "center", pb: "16px" }}
+                    >
+                      {pathwayCourseData.pathway}
+                    </Typography>
+                    <Typography variant="body1">
+                      {pathwayCourseData.description}
+                    </Typography>
+
+                    <ExternalLink
+                      style={{
+                        textDecoration: "none",
+                      }}
+                      href={"https://youtu.be/DDFvJmC3J5M"}
+                    >
+                      <Typography
+                        style={{ display: "flex" }}
+                        mt={2}
+                        align="start"
+                        variant="body2"
+                      >
+                        <img
+                          src={require("./asset/ComputerScreen.svg")}
+                          alt="MonitorScreen Img"
+                        />
+                        <section
+                          className={classes.link}
+                          onClick={handleVideo}
+                          style={{
+                            cursor: "pointer",
+                          }}
+                        >
+                          {"  "} &nbsp; &nbsp;
+                          <b>What's it all about?</b>
+                        </section>
```

Comment:
Avoid use a `section` for link text.  See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section for when it should be used.

Reply:
> Having difficulty recreating now, but somehow there were no batches and errors ensued due to this bad request:
> 
> > XHR failed loading: GET "https://dev-api.merakilearn.org//pathways/undefined/courses?courseType=json"
> 
> ![image](https://user-images.githubusercontent.com/9159008/175458835-86bc168e-7fcd-43f2-a8e9-d831a0b5f653.png)

must be any backend issue will disable it or add a message. Thankyou for reporting this issue